inst/pre-commit-config.yaml

```diff
@@ -14,6 +14,7 @@ repos:
     # -   id: roxygenize
     # -   id: use-tidy-description
     # -   id: deps-in-desc
+    # -   id: codemeta-description-updated
```

Comment:
is this really  a package hook? Otherwise, please move it up to `#any R proejct`

Reply:
Yes it is only for packages (or projects structured as packages)